store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -89,12 +89,11 @@ public final class VolumeCLI extends SoapCLI {
     private static final String NOT_ALLOWED_EXTERNAL = " is not allowed for external storetype";
     private static final String NOT_ALLOWED = " is not allowed for edit";
     private static final String INVALID_STORE_TYPE = "invalid storetype";
-    private static final String HELP_EXTERNAL_NAME = "  only name can be edited for external store volumes ";
     private static final String MISSING_ATTRS = " is missing";
     private static final String NOT_ALLOWED_ID = "id cannot be specified when adding a volume";
 
     private static final String H_STORE_TYPE = "Store type: internal or external";
-    private static final String H_STORAGE_TYPE = "Name of the store provider (S3, ObjectStore, OPENIO)";
+    private static final String H_STORAGE_TYPE = "Name of the store provider (S3 or OPENIO)";
```

Comment:
Its Storage Type not a Store providers

Reply:
Let's make it `Supported storage Types are S3 and OPENIO`

Reply:
done